pkg/asset/manifests/openshift.go

```diff
@@ -201,7 +209,11 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 		assetData["99_cloud-creds-secret.yaml"] = applyTemplateData(cloudCredsSecret.Files()[0].Data, templateData)
 		assetData["99_role-cloud-creds-secret-reader.yaml"] = applyTemplateData(roleCloudCredsSecretReader.Files()[0].Data, templateData)
 	case baremetaltypes.Name:
-		assetData["99_baremetal-provisioning-config.yaml"] = applyTemplateData(baremetalConfig.Files()[0].Data, *installConfig.Config.Platform.BareMetal)
+		bmTemplateData := baremetalTemplateData{
+			Baremetal:     installConfig.Config.Platform.BareMetal,
+			ProvisioningOSDownloadURL: string(*new(rhcos.Image)),
```

Comment:
This will just be an empty string, you can see how workers does it:


https://github.com/openshift/installer/blob/c47dffde52e56a111ab7ce3d7c429d597cb15a74/pkg/asset/machines/worker.go#L161

https://github.com/openshift/installer/blob/c47dffde52e56a111ab7ce3d7c429d597cb15a74/pkg/asset/machines/worker.go#L163

https://github.com/openshift/installer/blob/c47dffde52e56a111ab7ce3d7c429d597cb15a74/pkg/asset/machines/worker.go#L214

Reply:
Also, once you add this as a dependency, the graph in the docs needs to be updated, there's a command at the bottom of https://github.com/openshift/installer/blob/master/docs/design/assetgeneration.md#dependency-graph

Reply:
Ah OK I see.  Thanks.